lib/galaxy_test/base/api.py

```diff
@@ -58,8 +58,9 @@ def _request_celery_app(self, celery_session_app, celery_config):
             if os.environ.get("GALAXY_TEST_EXTERNAL") is None:
                 from galaxy.celery import celery_app
 
-                celery_app.fork_pool.stop()
-                celery_app.fork_pool.join(timeout=5)
+                if celery_app.fork_pool:
```

Comment:
This seems weird, are you bootstrapping your own instance or targeting an external one ?

Reply:
This just subclasses integration test case: https://github.com/galaxyproject/total-perspective-vortex/blob/main/tests/test_mapper_resubmit.py
No other setup as such. Is anything else required?